### PR TITLE
log SvgLoader: Enhance log message

### PR DESCRIPTION
**Pull request by JSUYA**
- Description :
 Change tag name "Doc" to "Svg"
 Fix "Mask" tag name.
 Add element to skip output (title)
 Add attribute to skip output (id, data-name, version, overflow=visible, xmlns*, xml:space)

### CODE REVIEW

File: src/loaders/svg/tvgXmlParser.cpp [hunk 1]
```diff
@@ -61,10 +61,63 @@ string simpleXmlNodeTypeToString(SvgNodeType type)
...
         default: return "Unknown";
     }
     return "Unknown";
 }
+
+bool isIgnoreUnsupportedLogElements(const char* tagName)
+{
+    const auto elementsNum = 1;
+    const char* const elements[] = { "title" };
+    const unsigned int elementsLength[] = { 5 };
```
Comment (hermet) [inline src/loaders/svg/tvgXmlParser.cpp:74]: this looks unnecessary as we can do like this:
strncmp(tagName, elements[i], strlen(elements[i])  ...

File: src/loaders/svg/tvgXmlParser.cpp [hunk 2]
```diff
@@ -61,10 +61,63 @@ string simpleXmlNodeTypeToString(SvgNodeType type)
...
+    const auto tagAttributeLength = strlen(tagAttribute);
+
+    for (unsigned int i = 0; i < attributesNum; ++i) {
+        if (!strncmp(tagAttribute, attributes[i].tag, attributes[i].tagSize < tagAttributeLength ? attributes[i].tagSize : tagAttributeLength)) {
+            if (attributes[i].value && tagValue) {
+                const auto tagValueLength = strlen(tagValue);
+                if (!strncmp(tagValue, attributes[i].value, attributes[i].valueSize < tagValueLength ? attributes[i].valueSize : tagValueLength)) {
+                    return true;
+                }
+                else continue;
```
Comment (hermet) [inline src/loaders/svg/tvgXmlParser.cpp:113]: please fix coding convention:
if {
  ...
} else  {
  ...
}

**Review by hermet — CHANGES_REQUESTED** [review]
Please check comments.

File: src/loaders/svg/tvgXmlParser.cpp (hunk 2 above)
Comment (JSUYA) [reply src/loaders/svg/tvgXmlParser.cpp]: Fixed code

File: src/loaders/svg/tvgXmlParser.cpp (hunk 1 above)
Comment (JSUYA) [reply src/loaders/svg/tvgXmlParser.cpp]: I changed the code.

**Comment by hermet** [comment]
https://github.com/Samsung/thorvg/issues/201

**Review by hermet — APPROVED** [review]

**Pull request merged**